qa/automation/CareerPortal.Tests.Scenarios/4. Sending CV/4.1.Sending CV.feature

```diff
@@ -0,0 +1,132 @@
+@Priority:Critical
+Scenario Outline: 4.1_01 Correct display of the cv submit form
+    Given user wanted to send CV
+    When user opens "<name of the page>" the page
+    Then user finds the form for sending CV
+	And the form includes "Name", "Email", "Message" fields
+	And the form includes "Attach CV" and "Send" buttons
+	And the form includes checkbox "I agree to the personal data processing"
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_02 Successful submission with filled "Name", "Email" fields, checkbox and attached CV
+    Given user founded the Send CV form on page "<name of the page>"
+    And user entered "<Name>", "<Email>", "<Attach CV>", "<Checkbox>" fields with foolowing combination of values:
+        | Name               | Email               | Attach CV   | Checkbox |
+        | Evgeniia T         | tikeeva_es@mail.ru  | cv.pdf 10Mb | True     |
+        | Евгения Т          | tikeeva1998@mail.ru | cv.doc 2Mb  | True     |
+        | Evgeniia T S       | tikeeva_es@mail.ru  | cv.docx 2Mb | True     |
+        | Ж'еня-Тикеева, мл. | tikeeva1998@mail.ru | cv.rtf 2Mb  | True     |
+	When user clicks "Send" button
+    Then user should see the message that CV was submitted
+
+    Examples: 
+		| name of the page                                               |
+		| https://career.quantori.com/ru                                 |
+		| https://career.quantori.com/ru/positions/cloud-python-engineer |
+
+@Type:Positive
+@Priority:Critical
+Scenario: 4.1_03 Successful submission with filled "Name", "Email", "Message" fields, checkbox and attached CV
+    Given user entered "<Name>", "<Email>", "<Message>", "<Attach CV>", "<Checkbox>" fields with foolowing combination of values:
+        | Name               | Email               | Message                    | Attach CV   | Checkbox |
+        | Evgeniia T         | tikeeva_es@mail.ru  | Please consider my resume. | cv.pdf 10Mb | True     |
+        | Евгения Т          | tikeeva1998@mail.ru | I will be a good worker.   | cv.doc 2Mb  | True     |
+        | Evgeniia T S       | tikeeva_es@mail.ru  | Please consider my resume. | cv.docx 2Mb | True     |
+        | Ж'еня-Тикеева, мл. | tikeeva1998@mail.ru | I will be a good worker.   | cv.rtf 2Mb  | True     |
+	When user clicks "Send" button
+    Then user should see the message that CV was submitted
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_04 User can attach a valid file
+    Given user filled the Send CV form
+    When user attaches a file in "<Extension>" format
+	And file size is less than or equal to 10 Mb
+	Then attached file name should be displayed under the "Attach CV" button
+
+	Examples:
+        |Extension|
+		|.pdf     |
+		|.doc     |
+		|.docx    |
+		|.rtf     | 
```

Comment:
Dublicate 4.1_03 scenario ( we already have verification for size and format of file "Attach CV")

Reply:
Thanks for your comment!) 
I'll fix it)

Reply:
Mistake "foolowing" in 22, 40, 52 rows